src/notify/app/cache.py

```diff
@@ -0,0 +1,3 @@
+from flask_caching import Cache
+
+cache = Cache(config={'CACHE_TYPE': 'SimpleCache'})
```

Comment:
are there some defaults around what triggers the cache invalidating/refreshing? guess it's very unlikely to be updated much in real life, but might be when we're testing locally